src/hooks/useToggle/index.ts

```diff
@@ -45,6 +46,7 @@ const useToggle = ({
 
   const registerContainerRef = useCallback(
     (ref: HTMLElement | null): void => {
+      currentElementRef.current = ref;
```

Comment:
What is this added for?

Reply:
It was added to get width/height of menu element, but now it's not needed so I'll remove it